<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pidSetup</class>
 <widget class="QWidget" name="pidSetup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>213</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>680</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_22">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_23">
       <item>
        <widget class="QGroupBox" name="roll_box">
         <property name="title">
          <string>Roll</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="topMargin">
           <number>1</number>
          </property>
          <property name="rightMargin">
           <number>1</number>
          </property>
          <property name="bottomMargin">
           <number>1</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="roll_kp_label">
              <property name="minimumSize">
               <size>
                <width>73</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>73</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>Kp:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="roll_kp_edit">
              <property name="minimum">
               <double>-1000.000000000000000</double>
              </property>
              <property name="maximum">
               <double>1000.000000000000000</double>
              </property>
              <property name="singleStep">
               <double>0.500000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="roll_ki_label">
              <property name="minimumSize">
               <size>
                <width>73</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>73</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>Ki:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="roll_ki_edit">
              <property name="minimum">
               <double>-1000.000000000000000</double>
              </property>
              <property name="singleStep">
               <double>0.500000000000000</double>
              </property>
              <property name="value">
               <double>0.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="roll_kd_label">
              <property name="minimumSize">
               <size>
                <width>73</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>73</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>Kd:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="roll_kd_edit">
              <property name="minimum">
               <double>-1000.000000000000000</double>
              </property>
              <property name="maximum">
               <double>1000.000000000000000</double>
              </property>
              <property name="singleStep">
               <double>0.500000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="label">
              <property name="minimumSize">
               <size>
                <width>73</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>73</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>Error to rate</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="roll_rate_edit"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QPushButton" name="roll_set_btn">
              <property name="text">
               <string>SET</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="roll_restore_btn">
              <property name="text">
               <string>RESTORE</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="pitch_box">
         <property name="title">
          <string>Pitch</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QLabel" name="pitch_kp_label">
              <property name="minimumSize">
               <size>
                <width>73</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>73</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>Kp:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="pitch_kp_edit">
              <property name="minimum">
               <double>-1000.000000000000000</double>
              </property>
              <property name="maximum">
               <double>1000.000000000000000</double>
              </property>
              <property name="singleStep">
               <double>0.500000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_9">
            <item>
             <widget class="QLabel" name="pitch_ki_label">
              <property name="minimumSize">
               <size>
                <width>73</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>73</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>Ki:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="pitch_ki_edit">
              <property name="minimum">
               <double>-1000.000000000000000</double>
              </property>
              <property name="singleStep">
               <double>0.500000000000000</double>
              </property>
              <property name="value">
               <double>0.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <widget class="QLabel" name="pitch_kd_label">
              <property name="minimumSize">
               <size>
                <width>73</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>73</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>Kd:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="pitch_kd_edit">
              <property name="minimum">
               <double>-1000.000000000000000</double>
              </property>
              <property name="maximum">
               <double>1000.000000000000000</double>
              </property>
              <property name="singleStep">
               <double>0.500000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_21">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="minimumSize">
               <size>
                <width>73</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>73</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>Error to rate</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="pitch_rate_edit"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_10">
            <item>
             <widget class="QPushButton" name="pitch_set_btn">
              <property name="text">
               <string>SET</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pitch_restore_btn">
              <property name="text">
               <string>RESTORE</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="yaw_box">
         <property name="title">
          <string>Yaw</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_11">
            <item>
             <widget class="QLabel" name="yaw_kp_label">
              <property name="minimumSize">
               <size>
                <width>73</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>73</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>Kp:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="yaw_kp_edit">
              <property name="minimum">
               <double>-1000.000000000000000</double>
              </property>
              <property name="maximum">
               <double>1000.000000000000000</double>
              </property>
              <property name="singleStep">
               <double>0.500000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_12">
            <item>
             <widget class="QLabel" name="yaw_ki_label">
              <property name="minimumSize">
               <size>
                <width>73</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>73</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>Ki:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="yaw_ki_edit">
              <property name="minimum">
               <double>-1000.000000000000000</double>
              </property>
              <property name="singleStep">
               <double>0.500000000000000</double>
              </property>
              <property name="value">
               <double>0.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_13">
            <item>
             <widget class="QLabel" name="yaw_kd_label">
              <property name="minimumSize">
               <size>
                <width>73</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>73</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>Kd:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="yaw_kd_edit">
              <property name="minimum">
               <double>-1000.000000000000000</double>
              </property>
              <property name="maximum">
               <double>1000.000000000000000</double>
              </property>
              <property name="singleStep">
               <double>0.500000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_19">
            <item>
             <widget class="QLabel" name="label_4">
              <property name="minimumSize">
               <size>
                <width>73</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>73</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>Error to rate</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="yaw_rate_edit"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_14">
            <item>
             <widget class="QPushButton" name="yaw_set_btn">
              <property name="text">
               <string>SET</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="yaw_restore_btn">
              <property name="text">
               <string>RESTORE</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="depth_box">
         <property name="title">
          <string>Depth</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_17">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="minimumSize">
               <size>
                <width>73</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>73</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>Kp:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="depth_kp_edit"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_15">
            <item>
             <widget class="QLabel" name="label_9">
              <property name="minimumSize">
               <size>
                <width>73</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>73</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>Ki:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="depth_ki_edit"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_16">
            <item>
             <widget class="QLabel" name="label_12">
              <property name="minimumSize">
               <size>
                <width>73</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>73</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>Kd:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="depth_kd_edit"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_20">
            <item>
             <widget class="QLabel" name="label_5">
              <property name="minimumSize">
               <size>
                <width>73</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>73</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>Error to rate</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="depth_rate_edit"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_18">
            <item>
             <widget class="QPushButton" name="depth_set_btn">
              <property name="text">
               <string>SET</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="depth_restore_btn">
              <property name="text">
               <string>RESTORE</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QPushButton" name="all_set_btn">
         <property name="text">
          <string>SET ALL</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="all_restore_btn">
         <property name="text">
          <string>RESTORE ALL</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
